src/containers/CreateDoc/index.tsx

```diff
@@ -1,26 +1,120 @@
-import React from 'react';
-import Dropdown from '../../components/Dropdown';
-import documentTypes from './config';
-import { createPdf } from './pdf';
+import React, { useState } from "react";
+import Dropdown from "../../components/Dropdown";
+import documentTypes from "./config";
+import { createPdf } from "./pdf";
+import "./createDoc.css";
+
+import Button from "@material-ui/core/Button";
+import GetAppIcon from "@material-ui/icons/GetApp";
+import VisibilityIcon from "@material-ui/icons/Visibility";
+import createDocStyles from "./styles";
+import DeleteIcon from "@material-ui/icons/Delete";
 
 const CreateDoc: React.FC = () => {
+  const [docProcessInitialized, handleProcessInitialization] = useState<
+    Boolean
```

Comment:
Why do we need this indentation here ?

Reply:
No need at all. Was using prettier till i got rid of it 🗡️ .
Premonition: A well-known remove unused code(indentation-ish) is coming

Reply:
haha